### PR TITLE
Add example build.fsx to readme

### PR DESCRIPTION
**Pull request by mglodack**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,46 @@ FAKE targets for getting projects off the ground ASAP
...
+
+```fsx
+#r @"packages/FAKE/tools/FakeLib.dll"
+#r @"packages/FSharp.FakeTargets/tools/FSharpFakeTargets.dll"
+
+open Fake
+
+let private _overrideConfig (parameters : datNET.Targets.ConfigParams) =
+  { parameters with
+      Project     = "Pigs Can Fly"
```
Comment (awseward) [inline README.md]: I feel like we should not encourage spaces in a project name... "PigsCanFly", maybe?

**Comment by awseward** [comment]
+1 despite comment

**Comment by mikeastock** [comment]
Can we get this out ASAP? :shipit:

**Comment by mikeastock** [comment]
Really could use this.

File: README.md (hunk 1 above)
Comment (mikeastock) [inline README.md]: I think this comment should be blocking

**Pull request merged**